repose-aggregator/components/filters/herp-filter/src/main/scala/org/openrepose/filters/herp/HerpFilter.scala

```diff
@@ -156,7 +156,7 @@ class HerpFilter @Inject()(configurationService: ConfigurationService,
       "responseCode" -> httpServletResponse.getStatus,
       "responseMessage" -> Try(HttpStatus.valueOf(httpServletResponse.getStatus).name).getOrElse("UNKNOWN"),
       "guid" -> Option(stripHeaderParams(TracingHeaderHelper.getTraceGuid(httpServletRequest.getHeader(CommonHttpHeader.TRACE_GUID.toString))))
-        .getOrElse(java.util.UUID.randomUUID.toString),
+        .getOrElse("").concat(":").concat(java.util.UUID.randomUUID.toString),
```

Comment:
I thought we decided to not lead with the colon if we didn't have anything? Maybe we explicitly say we couldn't find the transaction id? So something like `NO_TRANSACTION_ID:{guid}`?